central/graphql/resolvers/image_vulnerabilities.go

```diff
@@ -101,3 +230,313 @@ func withImageCveTypeFiltering(q string) string {
 	return search.AddRawQueriesAsConjunction(q,
 		search.NewQueryBuilder().AddExactMatches(search.CVEType, storage.CVE_IMAGE_CVE.String()).Query())
 }
+
+/*
+Sub Resolver Functions
+*/
+
+func (resolver *imageCVEResolver) EnvImpact(ctx context.Context) (float64, error) {
+	allCount, err := resolver.root.DeploymentCount(ctx, RawQuery{})
+	if err != nil || allCount == 0 {
+		return 0, err
+	}
+	scopedCount, err := resolver.root.DeploymentCount(resolver.withImageVulnerabilityScope(ctx), RawQuery{})
+	if err != nil {
+		return 0, err
+	}
+	return float64(scopedCount) / float64(allCount), nil
+}
+
+func (resolver *imageCVEResolver) FixedByVersion(ctx context.Context) (string, error) {
+	scope, hasScope := scoped.GetScope(ctx)
+	if !hasScope {
+		return "", nil
+	}
+	if scope.Level != v1.SearchCategory_IMAGE_COMPONENTS {
+		return "", nil
+	}
+
+	edgeID := postgres.IDFromPks([]string{scope.ID, resolver.data.GetId()})
+	edge, found, err := resolver.root.ComponentCVEEdgeDataStore.Get(ctx, edgeID)
+	if err != nil || !found {
+		return "", err
+	}
```

Comment:
Thought about this a little more. We can (should) remove dependency on `postgres.IDFromPks`. Instead search `ComponentCVEEdgeDataStore` for `Component ID=scope.ID AND CVE ID = resolver.data.GetId()`